### PR TITLE
chore: Change plan name Enterprise to Premium on marketplace

### PR DESCRIPTION
**Pull request by csuadev**
Because of the new NBJ, the Enterprise plan is going to be renamed to Premium, this introduce this change on the marketplace UI. 

## Proposed changes (including videos or screenshots)
image

## Issue(s)
[https://rocketchat.atlassian.net/browse/AECO-368](https://rocketchat.atlassian.net/browse/AECO-368)

### CODE REVIEW

File: apps/meteor/client/views/marketplace/components/MarketplaceHeader.tsx [hunk 1]
```diff
@@ -25,8 +25,11 @@ const MarketplaceHeader = ({ title }: { title: string }): ReactElement | null =>
 		return null;
 	}
 
+	// TODO NBJ: Remove this when enterprise is renamed to premium in the Apps data
+	const handleMarketplacePremiumTitle = title === t('Enterprise') ? t('Premium') : title;
```
Comment (tassoevan) [inline apps/meteor/client/views/marketplace/components/MarketplaceHeader.tsx:29]: Better handle this is the component above this one, avoiding match **translated** strings.

File: apps/meteor/client/views/marketplace/BundleChips.tsx [hunk 2]
```diff
@@ -18,17 +18,21 @@ const BundleChips = ({ bundledIn }: BundleChipsProps): ReactElement => {
...
-					title={t('this_app_is_included_with_subscription', {
-						bundleName: bundle.bundleName,
-					})}
-				>
-					{bundle.bundleName}
-				</Tag>
-			))}
+			{bundledIn.map(({ bundleId, bundleName }) => {
+				// TODO NBJ: Remove this when enterprise is renamed to premium in the Apps data
+				const handleBundleName = bundleName === 'Enterprise' ? 'Premium' : bundleName;
```
Comment (tassoevan) [inline apps/meteor/client/views/marketplace/BundleChips.tsx:23]: `question` Why not apply this "patch" on the REST endpoints that serve this information (`apps/meteor/ee/server/apps/communication/rest.ts`) instead?

**Review by tassoevan — CHANGES_REQUESTED** [review]

**Comment by hugocostadev** [comment]
maybe this is outdated... I guess that the Enterprise plan continue with the same name

**Review by AlenDavid — DISMISSED** [review]
LGTM!

**Review by tassoevan — APPROVED** [review]

**Review by AlenDavid — APPROVED** [review]

**Review by AlenDavid — APPROVED** [review]
LGTM

**Review by tassoevan — APPROVED** [review]

**Pull request merged**